jobs/spFilteringJob_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/data-preservation-programs/spade-tenant/db"
+)
+
+func TestCompare(t *testing.T) {
+	us := []string{"US"}
+
+	if !compare(us, "US", db.IncludedIn) {
+		t.Errorf("Expected %q to contain US", us)
+	}
+
+	if compare(us, "US", db.ExcludedFrom) {
+		t.Errorf("Expected %q to be excluded from contain US", us)
+	}
+
+	countries := []string{"AH", "CA", "US", "DE", "MX"}
```

Comment:
not part of this PR specifically, but we do need to think about / document what are the valid country/region/city codes that one would find in Kentik, so we can populate those correctly when applying criteria for a tenant. 

Initially it could be as simple as a spreadsheet with a big list of valid values, to make sure we pick from that list when populating the `EligibilityClauses`

Reply:
agreed. our validation story needs work

Reply:
Here is a [task](https://github.com/orgs/data-preservation-programs/projects/12/views/5?sliceBy%5Bvalue%5D=stephen-pl&pane=issue&itemId=53551792) to track that work